backend/spec/controllers/spree/admin/resource_controller_spec.rb

```diff
@@ -32,6 +32,13 @@ def model_class
     model do
       acts_as_list
       validates :name, presence: true
+      before_destroy :check_destroy_constraints
+
+      def check_destroy_constraints
+        return unless name == 'undestroyable'
+        errors.add :base, 'You cant destroy undestroyable things!'
```

Comment:
I'm not crazy about `You cant destroy undestroyable things!` I think there is a better way to phrase this.

We definitely need to add an apostrophe: `You can't destroy undestroyable things!`

Also, we probably want to add this to `en.yml` instead of just writing the string.

Reply:
This validation is only for testing as you can see. Adding it to en.yml is weird. But if you do not like the phrase itself, then you can offer absolutely any one to your taste.

Reply:
@bitberries That's fair, let's not worry about putting it in yaml then! I know it's picky, but I'd like to at least see the apostrophe :)

Reply:
@jacobherrington done